src/content/docs/en/guides/images.mdx

```diff
@@ -481,3 +480,95 @@ export default defineConfig({
 ## Community Integrations
 
 There are several third-party [community image integrations](https://astro.build/integrations?search=images) for optimizing and working with images in your Astro project.
+
+
+## Inline `.svg` files in the HTML (experimental)
```

Comment:
```suggestion
## Using `.svg` files as components (experimental)
```

Is this a more accurate title?

Reply:
Ooh, much better. (Though, on its own page, we won't need a heading at all. But rather a page title and description).

I'm just about to commit the standalone Experimental Flag PRs, so you'll see what we need.